prestoadmin/collect.py

```diff
@@ -209,10 +210,10 @@ def system_info():
 
 def get_system_info(download_location):
 
-    if not exists(TMP_PRESTO_DEBUG):
-        run("mkdir " + TMP_PRESTO_DEBUG)
+    if not exists(TMP_PRESTO_DEBUG_REMOTE):
```

Comment:
it's created by the user that's running presto-admin, which right now always is root since you have to run presto-admin as sudo.

collect system_info is the only of the "collect" commands that needs to write out a file when executing a command on the remote nodes, and that remote command is run as the username you specify for sudo (which may not be root)